### ISSUE TITLE
How to only build a specific package?

### ISSUE THREAD

**Issue by JunzheJosephZhu** [ethz-asl/kalibr#704]
Hi,
Im modifying some code under aslam_cv/aslam_cameras for debugging purposes.
If I run the build command
```catkin build -DCMAKE_BUILD_TYPE=Release -j4```
every time, it would be quite slow.
Is there a way to only build aslam_cv here?
Thanks,
Best,
Joseph

**Comment by goldbattle** [comment]
Yes, I recommend searching the documentation of `catkin build` as this is possible. It also depends if you want to build the dependencies.
